docs/secondary-network.md

```diff
@@ -68,10 +68,13 @@ data:
 ```
 
 At the moment, Antrea supports only a single OVS bridge for secondary networks,
-and supports up to eight physical interfaces on the bridge. The physical
-interfaces cannot be the Node's management interface, otherwise the Node's
-management network connectivity can be broken after `antrea-agent` creates the
-OVS bridge and moves the management interface to the bridge.
+and supports up to eight physical interfaces on the bridge.
+
+Note: When you set the Node's primary NIC as the secondary bridge physical interface,
```

Comment:
Should we mention DNS manager?

Reply:
Do you think the issue can happen with other managers too other than `systemd-resolved`?

Reply:
I think it's possible, I can double check other managers. Should we just use a general description `a DNS manager` instead of a specific one?

Reply:
How about "a DNS manager (e.g. system-resolved)"

Reply:
Updated.